### ISSUE TITLE
How to set up

### ISSUE THREAD

**Issue by black-snow** [activeadmin-plugins/active_admin_datetimepicker#35]
The usage section seems to be outdated. How do I correctly set this thing up?

Adding `//= require active_admin_datetimepicker` to `active_admin.js` yields

> Sprockets::FileNotFound
> couldn't find file 'active_admin_datetimepicker' with type 'application/javascript'

Adding `@import "active_admin_datetimepicker";` to `active_admin.scss` yields

> Sass::SyntaxError
> File to import not found or unreadable: active_admin_datetimepicker.

ruby 2.2.6, rails 4.2.9, aa 1.1.0

Found `config.register_javascript 'active_admin_datetimepicker.js'` in the issues - not mentioned in the README.

**Comment by workgena** [comment]
Thx for the bug-request. need to investigate this.

**Comment by workgena** [comment]
The #36 should fix this problem.

**Comment by black-snow** [comment]
Is there a release for that fix already?

**Comment by Fivell** [comment]
@black-snow   please try master branch, waiting for your feedback

**Comment by mikkotan** [comment]
@Fivell Do we need to install XDSoft DateTime Picker gem?
Cannot make `filter created_at, as: :date_time_range` to work.
Or do we need require jquery for this

**Comment by workgena** [comment]
No other requirements, it should just work

**Comment by workgena** [comment]
@michaelunltd check this demo fork(branch `dp_demo`; git clone; bundle install; rake db:setup; rails s)
https://github.com/workgena/demo.activeadmin.info/tree/dp_demo

Here you can see installation changes:
https://github.com/workgena/demo.activeadmin.info/commit/19509a3ae697c3cd8dae434613345ee9abd3c404

**Comment by mikkotan** [comment]
@workgena I'm trying to implement this in an existing project Rails 5.0.0 with activeadmin version `1.0.0.pre4`, the default datepicker is not working for me as well as this plugin. Any issues regarding `jquery-rails` in this specific version? Do you think this is `activeadmin` related issue?

Current Gem Versions:
activeadmin 1.0.0.pre4
jquery-rails 4.1.1

**Comment by grimmwerks** [comment]
Having somewhat of the same problem with Rails 5; thinking it might be the way datetimepicker inits itself  -- I'm not using turbolinks in my own site and I don't think the "on('page:load turbolinks:load', " isn't working.

**Comment by black-snow** [comment]
I feel like we're mixing different things up here.
I'm still on rails 4.x. IMHO 5.x issues should be in a separate issue.

When I opened this issue I wasn't sure whether this was a bug or just a configuration issue. Actually I'm still not quite sure. What I found in the readme didn't suffice and there were additional instructions hidden in other's issues. Sadly, nothing helped.
In the meantime I chose to use plain html5 pickers.

However, I'm willing to test fixes if you give me instructions on how to set this plugin up correctly :)

**Comment by Fivell** [comment]
@michaelunltd I think you should update from .pre version to stable one

**Comment by Fivell** [comment]
@black-snow   so this https://github.com/activeadmin-plugins/active_admin_datetimepicker/issues/35#issuecomment-333558615 is not working for you ?

**Comment by black-snow** [comment]
@Fivell I fear that I won't have time to get back to this anytime soon. Leaving this open for others having the same/similar issues.

**Comment by grimmwerks** [comment]
@Fivell I see you updated the javascript here:

https://github.com/activeadmin-plugins/active_admin_datetimepicker/blob/master/app/assets/javascripts/active_admin_datetimepicker.js.coffee

where you're checking document.ready or calling turbolinks.  I'm up to 0.5.0 of active_admin_datatimepicker and yet my javascript is the older one:

```
#= require jquery.xdan.datetimepicker.full

@setupDateTimePicker = (container) ->
  defaults = {
    formatDate: 'y-m-d',
    format: 'Y-m-d H:i',
    allowBlank: true,
    defaultSelect: false,
    validateOnBlur: false
  }

  entries = $(container).find('input.date-time-picker')
  entries.each (index, entry) ->
    options = $(entry).data 'datepicker-options'
    $(entry).datetimepicker $.extend(defaults, options)

$(document).on 'ready turbolinks:load', ->
  setupDateTimePicker $('body')
  $(document).on 'has_many_add:after', '.has_many_container', (e, fieldset) ->
    setupDateTimePicker fieldset
```

I've tried doing pristine, deleting and reinstalling etc and it seems I still have the older javascript:

(```
function() {
  this.setupDateTimePicker = function(container) {
    var defaults, entries;
    defaults = {
      formatDate: 'y-m-d',
      format: 'Y-m-d H:i',
      allowBlank: true,
      defaultSelect: false,
      validateOnBlur: false
    };
    entries = $(container).find('input.date-time-picker');
    return entries.each(function(index, entry) {
      var options;
      options = $(entry).data('datepicker-options');
      return $(entry).datetimepicker($.extend(defaults, options));
    });
  };

  $(document).on('ready turbolinks:load', function() {
    setupDateTimePicker($('body'));
    return $(document).on('has_many_add:after', '.has_many_container', function(e, fieldset) {
      return setupDateTimePicker(fieldset);
    });
  });

}).call(this);


```

Would love to get the datatimepicker reasonable again.

**Comment by Fivell** [comment]
@workgena  maybe you have any updates on this ?

**Comment by Fivell** [comment]
possible fixed by  #37

**Comment by workgena** [comment]
release 0.6.1 contains this fix #37 
does it work for @grimmwerks ?

**Comment by workgena** [comment]
Recent version 0.6.3 should be the right solution.
